crates/api_models/src/payments.rs

```diff
@@ -5316,14 +5359,18 @@ pub struct OrderDetailsWithAmount {
 
 impl masking::SerializableSecret for OrderDetailsWithAmount {}
 
-#[derive(Debug, Default, Eq, PartialEq, serde::Deserialize, serde::Serialize, Clone, ToSchema)]
+#[derive(Debug, Default, PartialEq, serde::Deserialize, serde::Serialize, Clone, ToSchema)]
 pub struct OrderDetails {
     /// Name of the product that is being purchased
     #[schema(max_length = 255, example = "shirt")]
     pub product_name: String,
     /// The quantity of the product to be purchased
     #[schema(example = 1)]
     pub quantity: u16,
+    /// tax rate applicable to the product
```

Comment:
we should not add amount related details here, please remove these. You can also remove this struct